### ISSUE TITLE
invalid disabled key in stanza

### ISSUE THREAD

**Issue by theCMack** [splunk/vscode-extension-splunk#18]
When I set the disabled key in my inputs.conf I get invalid key:

> Invalid key in stanza [script:///opt/splunk/etc/apps/ta-myscript/script.sh] in inputs.conf, line 89: disabled = 1.

```
[script:///opt/splunk/etc/apps/ta-myscript/script.sh]
sourcetype = mysourcetype
index = myindex
interval = 00 2 * * *
disabled = 1
```

**Comment by JasonConger** [comment]
Looks like you found a bug in the actual `inputs.conf.spec` file.  The Visual Studio Code extension reads the `.spec` files as you type.  Looking at the [inputs.conf.spec file](https://docs.splunk.com/Documentation/Splunk/latest/Admin/Inputsconf), there is no `disabled` setting for `[script://<cmd>]` stanzas or global settings.

You can "fix" this by editing the `inputs.conf.spec` file.  If you are using the extension's built-in `.spec` files, they are located in the [extension's directory ](https://code.visualstudio.com/docs/editor/extension-gallery#_where-are-extensions-installed) under `spec_files`.  Add the following right after the `GENERAL SETTINGS` section:

`disabled = [0|1]`

If you are [specifying a custom file path for the `.spec` files](https://github.com/splunk/vscode-extension-splunk/wiki/Extension-Settings#spec-file-settings) , you will need to update the `inputs.conf.spec` file in your custom file path.

I'll raise an issue on the Splunk side to correct the `.spec` file.

**Comment by mccarthyp-snet** [comment]
Not sure if you want a new Issue created for this, but, in addition to the `default` key, I'm getting:
> Invalid key in stanza [script://./bin/lsof.sh] in inputs.conf, line 2: interval = 600.
With this config:
```
[script://./bin/lsof.sh]
interval = 600
sourcetype = lsof
source = lsof
index = os
disabled = 0
```

**Comment by JasonConger** [comment]
Hi @mccarthyp-snet, this looks like a new issue.

**Comment by theCMack** [comment]
I was able to resolve both the disabled and interval false positives by adding them to the inputs.conf.spec as outlined by @JasonConger, Thanks!

**Comment by JasonConger** [comment]
Resolved in version 0.2.4
